src/java.base/unix/classes/java/lang/ProcessEnvironment.java

```diff
@@ -103,22 +112,22 @@
         } else {
             tracePrunedEnvVarsValue = 0;
         }
-        // We cache the C environment.  This means that subsequent calls
-        // to putenv/setenv from C will not be visible from Java code.
-        byte[][] environ = environ();
         origEnvironment = new HashMap<>(environ.length/2 + 3);
```

Comment:
Spaces around binary operators in new code.

Reply:
It may be derived from OpenJDK code, but it's an addition for OpenJ9 and so should follow OpenJ9 formatting guidelines.

Reply:
This is the extension repo primarily hosting OpenJDK code, iirc, you requested coding format matching existing formatting before.

Reply:
OpenJ9 has formatting guidelines?

Reply:
Many of the motivations behind [OMR's guidelines](https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md) are relevant to Java code and to here too. The specific point is
> Include spaces on both sides of most operators.

which "improves readability".

Yes, we strive to limit the differences relative to upstream, but here we're talking about additions that are highly unlikely to appear upstream.